azure-pipelines-dependency-graph-testing.yml

```diff
@@ -149,21 +149,21 @@ stages:
           - script: |
               # Install dependencies
               pip install pytest pytest-cov pytest-azurepipelines
+              git clone https://github.com/superconductive/dgtest.git
+              pip install -e dgtest
```

Comment:
This will eventually be moved to a proper `pip install dgtest` if we finalize this approach and upload to PyPi